### PR TITLE
CDPT-1995: TypeError: r(...) is not a function

### PR DESCRIPTION
**Pull request by EmilyHazlehurst**
- remove nselect.js script and all reference
- copy css for the select box but use default browser styles for the dropdown

### CODE REVIEW

File: public/app/themes/justice/src/css/global.css [hunk 1]
```diff
@@ -2309,89 +2303,37 @@ body.inner-cr .search-wrapper .search {
...
-
-.nselect li:hover, .nselect li.hover {
-    background: #f2f2f2;
-    text-decoration: underline;
-}
-
-.nselect li.option0 {
-    border-top: 1px solid #b3b2b2;
+    width: 162px;
+    &:after {
```
Comment (wilson1000) [inline public/app/themes/justice/src/css/global.css:2309]: Hey @EmilyHazlehurst , we have `sass` in `css` - did you mean this?

**Review by wilson1000 — COMMENTED** [review]
A comment on SASS usage

**Comment by EmilyHazlehurst** [comment]
> A comment on SASS usage

Thanks @wilson1000, I've moved the form styling into form.scss for consistency.

**Review by wilson1000 — APPROVED** [review]
Nice 👍

**Pull request merged**